example/Makefile

```diff
@@ -0,0 +1,13 @@
+.PHONY: dredd
+
+hooks/hooks:
+	cd hooks/; go build; cd ..;
+
+server:
+	go build -o server
+
+dredd: hooks/hooks server
+	~/Code/dredd/bin/dredd api.apib http://localhost:8080 --language go --server ./server --hookfiles hooks/hooks
```

Comment:
Does this need to be a hardcoded path? Won't work for most people if their code isn't in the same location.